libbeat/idxmgmt/client_handler.go

```diff
@@ -45,7 +45,7 @@ type ESClient interface {
 // prepare a policy and write alias.
 type FileClient interface {
 	GetVersion() common.Version
-	Write(name string, body string) error
+	Write(kind string, name string, body string) error
```

Comment:
what does 'kind' stand for? Which values can it have?

Reply:
It is used to create a subdirectory to write to, e.g. `template` or `policy`.

Reply:
Maybe add a comment for the parameters explaining their uses.

kind: resource type like template or policy
name: resource name
body: full contents of the resource to install

Maybe it would make sense to introduce a generic resource descriptor?

Reply:
How about naming it `component`? I have another PR waiting based on this one, that will introduce a type component, which I think would then be a good fit here.

Reply:
I'm fine with 'component'